### PR TITLE
Images optimisation

### PR DESCRIPTION
**Pull request by rosemcc**
Using Contentful image API to fetch webp images where possible, and adding srcset to use original image format as fallback
For images that are added as background images in CSS, we can detect if the browser supports webp then conditionally apply a webp image url or the original image url. 

Add OG tags (https://ogp.me/), Twitter tags

Clean up unused images

### CODE REVIEW

**Comment by rosemcc** [comment]
Note: Have not added webp format for images used as background images, because CSS doesn’t provide a built-in solution for fallback images that’s optimal. You can use multiple backgrounds image sources but that method can end up downloading both resources which would defeat the purpose of trying to use webp.

**Review by robmint — APPROVED** [review]

File: research-hub-web/src/app/components/articles/articles.component.html [hunk 1]
```diff
@@ -39,11 +39,10 @@
...
         <div>
-            <div class="banner-container" [style]="article.banner ? 
-                isMobile ?
-                'background: linear-gradient( rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)), url(' + article.banner.url + ') no-repeat; padding: 72px 0px' 
-                : 'background: linear-gradient( rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)), url(' + article.banner.url + ') no-repeat fixed center; padding: 72px 0px' 
-                : ''">
+            <div class="banner-container" 
+                [ngClass]="bannerImageUrl === undefined ? '' : isMobile ? 'mobile-banner' : 'standard-banner'" 
+                [ngStyle]="bannerImageUrl && {'background-image': 'url(' + bannerImageUrl + ')'}">
+
```
Comment (robmint) [inline research-hub-web/src/app/components/articles/articles.component.html:45]: URL param: w=1600

File: research-hub-web/src/app/components/home/research-activity/research-activity.component.html [hunk 2]
```diff
@@ -27,7 +27,10 @@ <h3>{{ item.name }}</h3>
...
             <!-- Panel Image -->
             <div fxHide.lt-md class="research-category-img">
-              <img src="{{ item.image?.url }}" alt="{{ item.name }}-image">
+                <img srcset="{{ item.image?.url }}?w=1000&fm=webp,
+                             {{ item.image?.url }}?w=1000"
+                     src="{{ item.image?.url }}?w=1000"
+                     alt="{{ item.name }}">
             </div>
           </div>
         </div>
```
Comment (robmint) [inline research-hub-web/src/app/components/home/research-activity/research-activity.component.html:36]: research activity

Param: w=1000

File: research-hub-web/src/app/components/shared/body-media/body-media.component.html [hunk 3]
```diff
@@ -5,8 +5,13 @@
...
-                    <a href="{{ contentItem.url }}"><img *ngIf="contentItem.url" src="{{ contentItem.url }}"
-                        alt="{{ contentItem.title }}-img"/></a><br>{{ contentItem.title }}
+                    <a href="{{ contentItem.url }}">
+                        <img *ngIf="contentItem.url" 
+                             srcset="{{ contentItem.url }}?fm=webp,
+                                     {{ contentItem.url }}"
+                             src="{{ contentItem.url }}"
+                             alt="{{ contentItem.title }}"/>
+                    </a><br>{{ contentItem.title }}
                     <small class="asset-summary" *ngIf="contentItem.description">&nbsp;—&nbsp;{{ contentItem.description }}</small>
```
Comment (robmint) [inline research-hub-web/src/app/components/shared/body-media/body-media.component.html:15]: URL param: w=900

**Comment by robmint** [comment]
All looks good and working as expected!

**Pull request merged**